### ISSUE TITLE
Units of Measure

### ISSUE THREAD

**Issue by fosterlynn** [lightningrodlabs/rea-playspace#38]
Units are a thing that it will be important to standardize on, to more easily communicate across networks.  VF uses the unit list from [OM2](https://github.com/HajoRijgersberg/OM).  I'm exploring what they have available (like they do have a couple unit conversion libraries), and also just extracting their list.

For now, we can just pick out what Code A and any other initial groups need and get them in manually.  Eventually, I can picture each group or network being able to select what they need from the big list (really big!  but categorized).  And then just using that list in their app.

I wonder if there should be a separate holochain app for units?  Or just a zome?

More thought and investigation needed.

**Comment by adaburrows** [comment]
I believe that pospi was pushing for a separate zome to be made. Right now, I've got a structure that maps onto both hREA and OM2. Also, the current units in the playspace are pulled straight from the OM2 spec (but without any of the metadata needed for conversions and so forth), but a separate OM2 zome would likely have all that capability. At some point, I'm going to expand the list to have more of the common units (my examples are getting more and more contrived because of the limitations).

**Comment by fosterlynn** [comment]
>At some point, I'm going to expand the list to have more of the common units (my examples are getting more and more contrived because of the limitations).

Yeah, I said I would do that for you, but am still in process, sorry about that.  I'm combining the smaller task with pulling them all out into an easier data structure(s), and also exploring what OM2 provides.  I think maybe what OM2 provides doesn't matter though, we'll need our own in holochain tech.  We can possibly use one of the OM2 libraries for conversions when that time comes though.

**Comment by fosterlynn** [comment]
Update: The file `units-en-useful.csv` is the current list of units.  The file and more explanation are here https://lab.allmende.io/valueflows/valueflows/-/tree/master/units.  We can help user groups pick units from that list for now, and load manually.  I'm closing this issue as good enough for now, but feel free to disagree and re-open!
